Makefile

```diff
@@ -60,9 +59,7 @@ lint: node_modules $(ESLINT) $(SRCS)
 # make nsp always pass - run this as separate travis task for "reporting"
 .PHONY: nsp
 nsp: node_modules $(NSP)
-	$(NPM) shrinkwrap --dev
```

Comment:
If you remove the shrinkwrap it won't check for issues with transitive module (which is typically where most security issues occur, at lower level modules). Any reason why we don't want to have that visibility?